app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ArticlesController < ApplicationController
+    def new
+        @pg_title = "Create Post | Haxxor News"
+        @article = Article.new
+    end
+    
+    def create
+        @article = Article.new(article_param)
+        
+        if @article.save
+            redirect_to :controller => 'home', :action => 'index'
+        else
+            render 'new'
+        end
+    end
+    
+    private
+    def article_param
```

Comment:
What do you think of using `article_params` or `permitted_params`? There's a couple of parameters you're accepting so I think a pluralized method name would fit better.